network-store-model/src/main/java/com/powsybl/network/store/model/HvdcLineAttributes.java

```diff
@@ -44,7 +44,7 @@
     private Map<String, String> aliasByType = new HashMap<>();
 
     @Schema(description = "Resistance")
-    private double r = Double.NaN;
+    private double r;
```

Comment:
For next attributes classes, you add NaN as init value and here you do the reverse. Why ?

Reply:
I don't initialize all double values to `Double.NaN` , I make homogenization between this code and the powsybl-core one.
For this example in `powsybl-core/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/HvdcLineImpl.java` :     
r, maxP, aren't initialized (0.0 by default), then I removed initialization to NaN in here.